src/profilers.hpp

```diff
@@ -16,6 +16,49 @@
 #include <unordered_set>
 
 namespace aperf {
+
+
+   /**
+     A class describing metric reader profiler.
+  */
+  class MetricReader : public Profiler{
+
+    private:
+      std::string metric_command;
+      std::string metric_name;
+      std::string regex;
+      int freq;
+      unsigned int server_buffer;
+      std::string name;
+      std::vector<std::unique_ptr<Requirement> > requirements;
+      fs::path result_out;
```

Comment:
What's the purpose of ```result_out``` here? I don't see it used anywhere in your code.

Reply:
Used for stderr (like it was for perf profiler)

Reply:
Removed it now